src/ImageCard.react.js

```diff
@@ -50,6 +50,14 @@ var ImageCard = React.createClass({
     }
     util.exec(['open', $repoUri + this.props.image.name]);
   },
+  componentDidMount: function() {
+    $.get('https://registry.hub.docker.com/v1/repositories/' + this.props.image.name + '/tags', function (result) {
+      this.setState({
+        tags: result,
+        chosenTag: result[0].name
+      });
+    }.bind(this));
```

Comment:
Since we have es6 support, you can use fat arrow to avoid needing .bind(this)

- Remove .bind(this)
- Change:

```
function (result) {
}
```

to

```
(result) => {
}
```